### PR TITLE
Adds rate-limiting for number of APIs can be created

### PR DESCRIPTION
**Pull request by Ketharan**
## Purpose
Fixes https://github.com/wso2-enterprise/choreo/issues/5974

### CODE REVIEW

**Comment by DMadhuranga** [comment]
\azp run

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/templates/repository/conf/api-manager.xml.j2 [hunk 1]
```diff
@@ -303,6 +303,15 @@
 
     </Analytics>
 
+    <!--
+        Configuration to enable rate-limiting the no of APIs can be created and set the limit
+    -->
+    <RateLimit>
```
Comment (praminda) [inline features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/templates/repository/conf/api-manager.xml.j2:309]: I think this should be ChoreoRateLimit

**Comment by Ketharan** [comment]
This is added to main branch and extension repo

**Pull request closed without merging**